removing excel_can_ac format key to address migration issue [sc-180121]

liamlynch-data opened the pull request.

liamlynch-data committed:
removing excel_can_ac format key to address migration issue [sc-180121]

liamlynch-data committed:
minor changes to some user facing text, changelog [sc-180121]

Sallsamba reviewed (approved):
LGTM!

MayeulRousselet reviewed (changes requested):
The plugin version number should be also updated in plugin.json

liamlynch-data commented:
> The plugin version number should be also updated in plugin.json

@MayeulRousselet the plugin version shouldn't change with this PR I think, it should stay on 1.0.1
Master is at 1.0.1 - which we have not released yet - and I added the changes we did here into the release notes of 1.0.1 (in addition to Arnaud's html conditional formating support). So hopefully it's ok as is.
